`package.json` is missing dependency: `shimmer`

The file [src/web-vitals-autoinstrumentation.ts#L39](https://github.com/honeycombio/honeycomb-opentelemetry-web/blob/main/src/web-vitals-autoinstrumentation.ts#L39) includes an import of [shimmer](https://www.npmjs.com/package/shimmer), but this is not defined as a dependency in [package.json](https://github.com/honeycombio/honeycomb-opentelemetry-web/blob/main/package.json).

I believe this was introduced in [v0.2.1](https://github.com/honeycombio/honeycomb-opentelemetry-web/releases/tag/v0.2.1) in #140.

**Versions**

0.2.1

**Steps to reproduce**

1. Run [depcheck](https://www.npmjs.com/package/depcheck) to detect unused/missing dependencies:

```shell
> npx depcheck

Unused devDependencies
* @types/jest
* jest
* jest-environment-jsdom
* prettier
* typescript
Missing dependencies
* shimmer: ./src/web-vitals-autoinstrumentation.ts
```

**Additional context**

This is reported by our build system (using esbuild + pnpm):

```shell
✘ [ERROR] Could not resolve "shimmer"

    node_modules/.aspect_rules_js/@honeycombio+opentelemetry-web@0.2.1/node_modules/@honeycombio/opentelemetry-web/dist/src/web-vitals-autoinstrumentation.js:41:37:
      41 │ const shimmer = __importStar(require("shimmer"));
         ╵                                      ~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "@opentelemetry/sdk-trace-web": "~1.24.0",
     "@opentelemetry/semantic-conventions": "~1.24.0",
     "axios": "^1.6.7",
+    "shimmer": "^1.2.1",
     "ua-parser-js": "^1.0.37",
     "web-vitals": "^3.5.2"
   }

